sdbuild/packages/pybind11/qemu.sh

```diff
@@ -1,7 +1,7 @@
 set -x
 set -e
 
-VERSION=2.5.0
+VERSION=2.6.0
```

Comment:
Pybind11 2.5 is required by Vitis runtime (VART) and I am not sure if 2.6 is compatible with that. So probably we still need 2.5.

Reply:
I thought it was related to PYNQ version.